### ISSUE TITLE
Special value that has no type: null in variables.tf

### ISSUE THREAD

**Issue by konstantin-recurly** [segmentio/terraform-docs#161]
### Prerequisites

Put an `x` into the box that applies:

- [ x] This issue describes a bug.
- [ ] This issue describes a feature request.

For more information, see the [Contributing Guidelines](https://github.com/segmentio/terraform-docs/tree/master/CONTRIBUTING.md).

### Description

Briefly describe the issue.

In [terraform](https://www.terraform.io/docs/configuration/expressions.html#null) variables it's possible to specify a value that represents absence or omission. If you set an argument of a resource or module to null, Terraform behaves as though you had completely omitted it. 

I would expect that terraform-docs still would be able to generate readme.me

#### Actual Behavior

Briefly describe what happened.
run terraform-docs
```
terraform-docs markdown .  > some.md
2020/01/14 13:51:01 At 18:47: Unknown token: 18:47 IDENT length
```

Tried to use a docker image
```
docker run --rm \
  -v $(pwd):/data \
  cytopia/terraform-docs \
  terraform-docs-replace-012 --sort-inputs-by-required --with-aggregate-type-defaults md some.md
```
Here it provides more logs
```Bash
terraform-docs-012 --sort-inputs-by-required --with-aggregate-type-defaults md /data
2020/01/14 18:55:47 At 250:26: Unknown token: 250:26 IDENT null

...
   245	variable "backup_configuration" {
   246	  description = "The backup_configuration settings subblock for the database setings"
   247	  type = "object"
   248
   249	  default = {
   250	    binary_log_enabled = null
   251	    enabled            = null
   252	    start_time         = null
   253	  }
   254
   255	}
...
```

Do you have long logs to share? Please go to https://ghostbin.com and insert the link here.

#### Expected Behavior

Briefly describe what you expected to happen.
The expected behavior is to generate some.md file.

### Replication Case

Tell us which steps to take to replicate your problem.

1. Create variables.tf
```
cat <<EOF > variables.tf
variable "backup_configuration" {
  description = "The backup_configuration settings"
  type = object({
    binary_log_enabled = bool
    enabled            = bool
    start_time         = string
  })
  default = {
    binary_log_enabled = null
    enabled            = null
    start_time         = null
  }
}
EOF
```
2. Run `terraform-docs`
```Bash
terraform-docs markdown .  > some.md
```

### Version
```Bash
terraform-docs version
terraform-docs version v0.0.0- v0.7.0 darwin/amd64 BuildDate: 2019-12-15T00:53:30+0000
```

**Comment by khos2ow** [comment]
This is fixed as part of supporting Terraform v0.12.x, by the look of provided version you're using an older version of the `terraform-docs` binary which didn't support this. Please get the latest RC binary from [release page](https://github.com/segmentio/terraform-docs/releases/tag/v0.8.0-rc.2) and try again.

Also note that the docker image in your example is unaffiliated with `terraform-docs`, make sure to contact with the maintainer of that docker image to have the latest available RC.

**Comment by konstantin-recurly** [comment]
Thanks. The latest release helped.
